packages/fxa-content-server/app/scripts/views/mixins/signin-mixin.js

```diff
@@ -170,29 +170,32 @@ export default {
       var verificationReason = account.get('verificationReason');
 
       if (
-        verificationReason === VerificationReasons.SIGN_IN &&
-        verificationMethod === VerificationMethods.EMAIL
+        (verificationReason === VerificationReasons.SIGN_IN &&
+          verificationMethod === VerificationMethods.EMAIL) ||
+        (verificationReason === VerificationReasons.SIGN_IN &&
+          verificationMethod === VerificationMethods.EMAIL) ||
```

Comment:
Unless my eyes are deceiving me, `(verificationReason === VerificationReasons.SIGN_IN && verificationMethod === VerificationMethods.EMAIL)` is repeated.

Reply:
`(verificationReason === VerificationReasons.SIGN_IN && verificationMethod === VerificationMethods.EMAIL)` is still the first and second grouped comparisons.